eras/babbage/formal-spec/remove-overlay.tex

```diff
@@ -155,8 +155,7 @@ \section{Removal of the Overlay Schedule}
   \begin{equation}\label{eq:prtcl}
     \inference[PRTCL]
     {
-      \var{bhb}\leteq\bheader{bh} &
```

Comment:
The definition of $bhb$ should stay since it's used in the call to $\mathsf{praosVrfChecks}$ below.